ssh/session.go

```diff
@@ -110,6 +111,15 @@ func NewSession(target string, session sshserver.Session) (*Session, error) {
 	s.Target = uid
 	s.Lookup = lookup
 
+	var connectionSource string
+
+	if _, ok := IsWs[uid]; ok {
+		connectionSource = "webterminal"
+		delete(IsWs, uid)
+	} else {
+		connectionSource = "ptyclient"
```

Comment:
please use 'pty'

Reply:
also, please use `const` for both so it is easier to find it in code and there is no risk of typos.